manage.py

```diff
@@ -373,25 +294,65 @@ def update_all(processes=1):
     """Update all derived data. Warning: Extremely slow on production data.
     """
     processes = int(processes)
-    update_functions(processes=processes)
-    load_districts()
     load_pacronyms()
     load_nicknames()
     load_election_dates()
-    update_itemized('a')
-    update_itemized('b')
-    update_itemized('e')
-    partition_itemized('a')
-    partition_itemized('b')
-    rebuild_aggregates(processes=processes)
     update_schemas(processes=processes)
```

Comment:
`update_schemas` is removed above, so the call should be removed here as well, right?

In fact, what should be in place for the removed methods here if we need to do a complete refresh?  I don't see a means of rebuilding the API-managed relations and objects like we had previously; do we need to re-run the migrations or a subset of them, or is there another way we should do this?  Or did I miss something entirely?